src/core/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ target_include_directories(smf
   SYSTEM PUBLIC ${PROJECT_SOURCE_DIR}/src/third_party/flat_hash_map
   SYSTEM PUBLIC ${PROJECT_SOURCE_DIR}/src/third_party/zstd/lib
   SYSTEM PUBLIC ${PROJECT_SOURCE_DIR}/src/third_party/xxHash
-  SYSTEM PUBLIC "${seastar_INCLUDE_DIRS}"
+  SYSTEM PUBLIC ${PROJECT_SOURCE_DIR}/src/third_party/seastar/include/seastar
```

Comment:
Nice. I think we'll need to port the `#include <core/foo.hh> -> #include <seastar/core/foo.hh>` Surprised this works.

Reply:
hmm, why? they just nested everything down a level